src/js/utils/LocalStorageUtil.ts

```diff
@@ -0,0 +1,17 @@
+const LocalStorageUtil = {
+  set(_key: string, _value: unknown) {
+    return (global as any).localStorage.setItem.apply(
```

Comment:
can we do without the type cast?

maybe you like one of these options:
* extend global to know about the builtin `WindowLocalStorage`-type.
* use `window` instead of `global`. the type-hassle should vanish.

Reply:
done

Reply:
much better! thanks for following up! 😍